libraries/networking/src/NetworkingConstants.h

```diff
@@ -25,9 +25,8 @@ namespace NetworkingConstants {
     // if you manually generate a personal access token for the domains scope
     // at https://staging.highfidelity.com/user/tokens/new?for_domain_server=true
 
```

Comment:
Code comments above need addressing / updating?

Reply:
I don't think at this point, it might be helpful to preserve these comments for a while for Project Apollo. And if we do remove/update them I'd suggest a separate PR for that.